cmd/process-agent/main_common.go

```diff
@@ -163,8 +164,11 @@ func runAgent(exit chan struct{}) {
 		return
 	}
 
-	// Run a profile server.
+	// Run a profile & telemetry server.
 	go func() {
+		if ddconfig.Datadog.GetBool("telemetry.enabled") {
```

Comment:
small nit: should we put the if statement outside the `go func() {}` closure to leave it more consistent with the main agent bootstrap?